kotlin/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'looker-kotlin-sdk'
```

Comment:
Good to know. This should help streamline the Kotlin SDK package creation. Funny it's in `settings.gradle` rather than `build.gradle` but I guess `settings` is used for processing `build`?

Reply:
I don't know why they're in separate locations.  I tried putting them both in settings and gradle hated that and I found other examples suggesting this is the way to do it, so here we are.